### DISCUSSION TITLE
win app sent photos received file

### DISCUSSION THREAD

**Discussion by FatBober** [python-telegram-bot/python-telegram-bot#3411] [Q&A]
win app 4.4x64  sent photos received file. send photos from android get a photo.
need help!

**Comment by Bibo-Joshi** [comment]
Hey. In order for someone to be able to help you, you must ask a good technical question. Please read [this short article]()https://github.com/python-telegram-bot/python-telegram-bot/wiki/Ask-Right and try again ;)
